eng/pipelines/release-go-start-pipeline.yml

```diff
@@ -9,7 +9,15 @@ parameters:
       (-fips) if they apply. Write in YAML format: write one version number per line with '-' and a
       space at the beginning of each line.
     type: object
-
+  - name: isSecurityRelease
+    displayName: >
+      This release includes security fixes:
+    type: string
+    default: Cancel
+    values:
+      - True
+      - False
+      - Cancel
```

Comment:
I'd prefer to keep a blank line above and below this param for readability.

Reply:
Fixed! 👍